endersgame/widgets/accounting.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+import time
+from IPython.display import display
+import ipywidgets as widgets
+from typing import Dict
+import warnings
+from endersgame.accounting.pnl import Pnl
+from endersgame.widgets.streams import StreamPoint, Prediction
+
+# Suppress FutureWarning
+warnings.simplefilter(action='ignore', category=FutureWarning)
+
+custom_css = """
+<style>
+    .table { 
+        width: 100%; 
+        margin-bottom: 1rem; 
+        background-color: #343a40; /* Dark grey background */
+        color: #f8f9fa; /* Light font color */
+        border-collapse: collapse;
+    }
+    .table th, .table td {
+        padding: 0.75rem;
+        vertical-align: top;
+        border-top: 1px solid #495057; /* Slightly lighter border */
+    }
+    .table thead th {
+        vertical-align: bottom;
+        border-bottom: 2px solid #495057; /* Slightly lighter header border */
+        background-color: #495057; /* Darker grey for header */
+        color: #f8f9fa; /* Light header text color */
+    }
+    .table tbody + tbody {
+        border-top: 2px solid #495057; /* Slightly lighter bottom border */
+    }
+    .table-striped tbody tr:nth-of-type(odd) {
+        background-color: #495057; /* Alternate row background */
+    }
+</style>
+"""
+
+# Dictionary mapping original column names to display names
+display_names = {
+    "stream_id": "Stream ID",
+    "current_ndx": "Current Index",
+    "num_resolved_decisions": "Resolved Decisions",
+    "total_profit": "Total Profit",
+    "wins": "Wins",
+    "losses": "Losses",
+    "win_loss_ratio": "Win/Loss Ratio",
+    "profit_per_decision": "Profit/Decision",
+    "standardized_profit_per_decision": "Standardized Profit/Decision",
+    "average_profit_per_decision": "Average Profit/Decision",
+    "avg_profit_per_decision_std_ratio": "Avg Profit/Decision Std Ratio"
+}
+
+class AccountingDataVisualizer:
+    def __init__(self):
+        self.account_model = Pnl
+        self.accountants = {}
+        self.df = pd.DataFrame(columns=[
+            "stream_id", "current_ndx", "num_resolved_decisions", "total_profit",
+            "wins", "losses", "win_loss_ratio", "profit_per_decision", "standardized_profit_per_decision"
+        ])
+        self.output = widgets.Output()
+        self.table_widget = widgets.HTML(value=self.data())
```

Comment:
question: will this works in non-jupyter environments?

Reply:
this is why I put things in `widget`, not sure what the behavior we want for other way to run things.

Reply:
this is an accounting widget to be shown in Notebook, so everything in the widget directory is meant for notebooks

Reply:
Will it be the repo with the Quickstarters or it will go in another repo?